pages/edit/flash/index.tsx

```diff
@@ -76,8 +76,7 @@ const FlashEditPage = () => {
             label: formData?.flashTimingType,
             value: formData?.flashTimingType,
           },
-          startDate: dayjs(formData?.activityStartDate).format('YYYY.MM.DD'),
-          endDate: dayjs(formData?.activityEndDate).format('YYYY.MM.DD'),
+          dateRange: [formatCalendarDate(formData?.activityStartDate), formatCalendarDate(formData?.activityEndDate)],
```

Comment:
```suggestion
          dateRange: {
  start: formatCalendarDate(formData?.activityStartDate),
  end: formatCalendarDate(formData?.activityEndDate)
}
```
전체적으로 코드에서 `dateRange[0]` `dateRange[1]` 의 형태로 사용되고 있는데, 다른 개발자가 코드를 읽을 때 배열의 각 요소가 어떤 의미인지 추가로 확인해야하는 불편함을 겪을 수 있을 것 같아요!
인덱스를 기반으로 하는 접근 방식은 타입 시스템이 감지하지 못 하기 때문에, 바꿔쓰는 실수가 발생할 수도 있고 확장성이나 코드 가독성 측면에서도 매직 넘버를 사용하는 것은 지양해야한다는 생각이 듭니다!
위와 같이 객체 형태로 사용하는 것에 대해서 어떻게 생각하시는지 궁금합니당

Reply:
헛,, 저도 배열로 정말 구현하기 싫었는데 
react 캘린더에 범위 기능을 활성화 하려면 값이 배열로 들어가야 합니다.
아마 모두 다 배열 => 객체로 변동 한다고 하면 공수가 꽤 될 것으로 예상되는데 ,, 
이번에 우선 배포 하고, 추후 리펙토링 하는 방향으로 생각해야 할 것 같아요
어떻게 생각하시나요 🥺

Reply:
react 캘린더에 배열로 들어가야한다면 `[dateRange.startDate, dateRange.endDate]` 이렇게 넣는게 좋을 것 같아용!
어차피 다음 QA 일요일로 잡혔구, 그 전까지 할당된 태스크 없으시니까 요 작업 하면 어떨까용?
dateRange[0] -> dateRange.startDate
dateRange[1] -> dateRange.endDate
vscode 이용해서 일괄적으로 바꾸면 편할 것 같습니다!